src/applications/managing-app-lifecycle.md

```diff
@@ -98,6 +98,22 @@ From deis-controller.local:peachy-waxworks
  * [new branch]      master     -> deis/master
 ```
 
+
+## Application Maintenance
+
+Maintenance mode for applications is useful to perform certain migrations or upgrades during which we don't want to serve client requests. Deis Workflow supports maintenance mode for an app during which the access to the app is blocked. Blocking access to the application means all the requests to the app are served with an error code of `503` and a static maintenance page by the router but the app will still be running and one-off commands can still be run.
```

Comment:
I feel it's ambiguous enough that a person would go looking in the router how to change it